src/MoBi.Presentation/Presenter/Main/NotificationPresenter.cs

```diff
@@ -176,20 +169,11 @@ public void ExportToFile()
          dataTable.ExportToCSV(file);
       }
 
-      private bool isShowing(NotificationType notificationType)
-      {
-         return VisibleNotification.Is(notificationType);
-      }
+      private bool isShowing(NotificationType notificationType) => VisibleNotification.Is(notificationType);
 
-      public void Toggle(NotificationType typeToToggle)
-      {
-         VisibleNotification = VisibleNotification ^ typeToToggle;
-      }
+      public void Toggle(NotificationType typeToToggle) => VisibleNotification ^= typeToToggle;
 
-      public void Handle(ShowValidationResultsEvent validationResultsEvent)
-      {
-         updateNotificationWith(validationResultsEvent.ValidationResult.Messages.Select(m => _notificationMessageMapper.MapFrom(m)));
-      }
+      public void Handle(ShowValidationResultsEvent validationResultsEvent) => updateNotificationWith(validationResultsEvent.ValidationResult.Messages.Select(m => _notificationMessageMapper.MapFrom(m)));
```

Comment:
I think this is where some { } are required , or at least breaking somewhere else. Right now the actual code is broken up in a weird place which is hard to read